spec/lib/notification_manager_spec.rb

```diff
@@ -66,17 +66,13 @@
       described_class.error(exception, message, ScienceWireClient.new)
     end
   end
-  context '.wos_logger' do
+  context 'WebOfScience.logger' do
     let(:wos_client) { WebOfScience::Client.new(Settings.WOS.AUTH_CODE) }
 
-    before do
-      described_class.class_variable_set(:@@wos_logger, nil)
-    end
-    it 'creates a single logger' do
-      expect(Logger).to receive(:new).with(Settings.WOS.LOG).once
-      described_class.error(exception, message, wos_client)
-    end
+    before { WebOfScience.class_variable_set(:@@logger, nil) }
+
     it 'logs errors' do
+      expect(WebOfScience).to receive(:logger).and_return(null_logger)
       expect(null_logger).to receive(:error).exactly(3)
       described_class.error(exception, message, wos_client)
     end
```

Comment:
This seems to have changed the meaning of the spec.  It's no longer testing that the logger is a single instance of a Logger, let's put that back.  I don't know why calling the `NotificationManager.error` once should be logging 3 errors, that seems weird; shouldn't it log one error?

Reply:
It isn't the `NotificationManager`'s concern anymore.  The responsibility is with the module.